plugins/modules/purefa_ds.py

```diff
@@ -435,7 +435,7 @@ def update_ds_v6(module, array):
         ds_change = True
     if module.params["bind_user"] != "":
         bind_user = module.params["bind_user"]
```

Comment:
It looks to me that the actual fix required is to just move line 437 into the `if` block starting on 438. No need for the change to the `if` statement 

Please try that.

Reply:
I agree. My original code needlessly sets the variable to what it already is in the case that the `uri` parameter was set to the same thing as what the array was set to.